statler_site/templates/play-detail-page.html

```diff
@@ -64,6 +66,11 @@
       <div id="details"></div>
       <div style="height: 200px; border-style: solid;">
         Review post box will go here.
+        <form action="{% url 'api:play-reviews' play_id %}" method="post">
+          {% csrf_token %}
```

Comment:
Form encoding is a standard way of sending data, and isn't messy or implementation-specific.

Also, the server-side code is just `request.POST['text']`. No deserialization to worry about: it is all done by django.

Granted, it is inconsistent with the JSON used for GETs. But I asked Anthony, and he wants to do it this way too.